docs/index.html

```diff
@@ -59,40 +59,40 @@ <h1 id="header-1">Features</h1>
             </dd>
 
-            <dt>A collection of static properties for often used token groups.</dt>
+            <dt>A collection of static properties for often-used token groups.</dt>
             <dd>
-                <p>Collections of related tokens as often used and needed for sniffs.</br>
+                <p>Collections of related tokens often used and needed for sniffs.</br>
         These are additional "token groups" to compliment the ones available through the PHPCS native <span><code>PHP_CodeSniffer\Util\Tokens</code></span> class.</p>
             </dd>
 
-            <dt>An ever growing number of utility functions for use with PHP_CodeSniffer.</dt>
+            <dt>An ever-growing number of utility functions for use with PHP_CodeSniffer.</dt>
             <dd>
-                <p>Whether you need to split an <span><code>array</code></span> into the individual items, are trying to determine which variables are being assigned to in a <span><code>list()</code></span> or are figuring out whether a function has a docblock, PHPCSUtils got you covered!</p>
+                <p>Whether you need to split an <span><code>array</code></span> into the individual items, are trying to determine which variables are being assigned to in a <span><code>list()</code></span> or are figuring out whether a function has a DocBlock, PHPCSUtils has you covered!</p>
 
-                <p>Includes improved versions of the PHPCS native utility functions and plenty new utility functions.</p>
+                <p>Includes improved versions of the PHPCS native utility functions and plenty of new utility functions.</p>
 
                 <p>These functions are, of course, compatible with PHPCS 2.6.0 up to PHPCS <span><code>master</code></span>.</p>
             </dd>
 
             <dt>Test utilities</dt>
             <dd>
-                <p>An abstract <span><code>UtilityMethodTestCase</code></span> class to allow for testing your own utility methods written for PHP_CodeSniffer with ease.</br>
+                <p>An abstract <span><code>UtilityMethodTestCase</code></span> class to allow for easier testing of your utility methods written for PHP_CodeSniffer.</br>
                 Compatible with both PHPCS 2.x as well as 3.x. Supports PHPUnit 4.x up to 9.x.</p>
             </dd>
 
             <dt>Backward compatibility layer</dt>
             <dd>
-                <p>A <span><code>PHPCS23Utils</code></span> standard which allows sniffs to work in both PHPCS 2.x and 3.x, as well as a number of helper functions for external standards which still want to support both PHP_CodeSniffer 2.x as well as 3.x.</p>
+                <p>A <span><code>PHPCS23Utils</code></span> standard which allows sniffs to work in both PHPCS 2.x and 3.x, as well as many helper functions for external standards which still want to support both PHP_CodeSniffer 2.x as well as 3.x.</p>
```

Comment:
This is specifically about the `PHPCSUtils\BackCompat\Helper` class which doesn't actually contain that many methods.

Reply:
I'll change to "a few".